### PR TITLE
feat(editor): Migrate copyPaste mixin to composables (no-changelog)

### PR DESCRIPTION
**Pull request by alexgrozav**
## Summary
Tech debt from Vue 3 migration. Required as part of NodeView refactoring.

## Related tickets and issues
https://linear.app/n8n/issue/N8N-6343/copypaste

## Review / Merge checklist
- [x] PR title and summary are descriptive. **Remember, the title automatically goes into the changelog. Use `(no-changelog)` otherwise.** ([conventions](https://github.com/n8n-io/n8n/blob/master/.github/pull_request_title_conventions.md))
- [x] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included.
   > A bug is not considered fixed, unless a test is added to prevent it from happening again.
   > A feature is not complete without tests.

### CODE REVIEW

File: packages/editor-ui/src/composables/useCopyPaste.ts [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+	navigator.appVersion.search('Chrome') === -1 &&
+	navigator.appVersion.search('CrMo') === -1 &&
+	navigator.appVersion.search('CriOS') === -1;
+const isInternetExplorer =
+	navigator.userAgent.toLowerCase().indexOf('msie') !== -1 ||
+	navigator.userAgent.toLowerCase().indexOf('trident') !== -1;
+
+type ClipboardEventFn = (data: string, event?: ClipboardEvent) => void;
+
+export function useCopyPaste(
```
Comment (netroy) [inline packages/editor-ui/src/composables/useCopyPaste.ts:19]: Can we rename this to `useClipboard` instead?

File: packages/editor-ui/src/composables/useCopyPaste.ts [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+		hiddenInputRef.value.select();
+	}
+
+	/**
+	 * Copies given data to clipboard
+	 *
+	 * @source https://hackernoon.com/copying-text-to-clipboard-with-javascript-df4d4988697f
+	 * @param value
+	 */
+	function copyToClipboard(value: string = ''): void {
```
Comment (netroy) [inline packages/editor-ui/src/composables/useCopyPaste.ts:74]: and rename this to just `copy`

**Review by netroy — COMMENTED** [review]
Could we use `useClipboard` from `vueuse` instead? That way we'd have less code to maintain.

File: packages/editor-ui/src/composables/useCopyPaste.ts (hunk 1 above)
Comment (alexgrozav) [reply packages/editor-ui/src/composables/useCopyPaste.ts]: Done, great suggestion!

File: packages/editor-ui/src/composables/useCopyPaste.ts (hunk 2 above)
Comment (alexgrozav) [reply packages/editor-ui/src/composables/useCopyPaste.ts]: Done.

**Comment by alexgrozav** [comment]
> Could we use `useClipboard` from `vueuse` instead? That way we'd have less code to maintain.

Good idea. I've made use of `useClipboard` for the copy part and simplified the `paste` part since we no longer support IE due to the Vue 3 upgrade anyway.

**Review by netroy — DISMISSED** [review]
Tested most of these manually just to be extra sure, and everything still works as expected.
Thanks for taking care of this 🙏🏽

**Review by netroy — DISMISSED** [review]

**Review by netroy — APPROVED** [review]

**Comment by Joffcom** [comment]
Got released with [`n8n@1.23.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.23.0)

**Pull request merged**